python/src/MorrisExperimentLHS_doc.i.in

```diff
@@ -3,9 +3,9 @@
 
 Available constructors:
 
-    MorrisExperiment(lhsDesign, N)
+    MorrisExperimentLHS(lhsDesign, N)
 
-    MorrisExperiment(lhsDesign, interval, N)
+    MorrisExperimentLHS(lhsDesign, interval, N)
```

Comment:
I think the MorrisExperiment interface class is not necessary in the first place.

Reply:
@jschueller yes and no. The implementation class factorizes some attribut & methods, checks... 
It dervies from WeightedExperimentImplementation but I don't know if it is a good idea?
Btw, the two **main** experiment classes (MorrisExperimentGrid and MorrisExperimentLHS) have similarities : only the first point (starting point) selection differs. 
Maybe we could think later about actorizing the classes?

Reply:
Ok I see, so the interface can still be removed. It's totally ok to inherit WeightedExperimentImplementation.

Reply:
ok It will be removed in a different PR